tempesta_fw/http_frame.c

```diff
@@ -1565,7 +1759,7 @@ tfw_h2_frame_process(void *c, TfwFsmData *data)
 		ss_skb_queue_purge(&h2->skb_head);
 	}
 
-	tfw_h2_context_reinit(h2, r == T_POSTPONE);
+	tfw_h2_context_reinit(h2, postponed);
```

Comment:
I'm wondering why can we receive postponed (partial) service frame? A comment for `tfw_h2_context_reinit()` doesn't describe the case.

Reply:
Yeah, the comment is somewhat incomplete - I'll correct that. In fact the `postponed` flag here is intended only for app frames; the service frames cannot be passed here as postponed, and the context is always reinitialized in `tfw_h2_context_reinit()` in case of service frames.
I added the possibility for upstairs processing for postponed app frames - to avoid the introduction of additional buffering level while processing the incoming request.

Reply:
Changed comment.